### PR TITLE
Change Node3D Gizmo to call group in realtime.

### PR DESCRIPTION
**Pull request by lyuma**
Averts godot crash which occurs if a node is added and then removed from the edited scene within one frame.

I have not filed an issue because the project and reproduction steps are complex, but essentially there is a tool script which adds nodes to the screen, manually ticks some IK processing on those nodes, then subsequently deletes them and saves out a PackedScene, all at once. The crash then happens at the next frame when the editor becomes responsive again.

Visual Studio stacktrace screenshot before applying this fix:
image
Basically, `call_group_flags` invokes a `Node3DEditor::_request_gizmo` with the default flags (0) so that the gizmo is called from MessageQueue on the next frame, after the node in question was destroyed.

Because the node was added to the active scene, it qualified for `is_node_being_edited`.

CC @TokageItLab

### CODE REVIEW

**Comment by lyuma** [comment]
Would be obsoleted by #51591

**Comment by Calinou** [comment]
@lyuma Do you think this change relevant for `3.x` as well? It's probably too risky to apply to 3.5 by now, but maybe it could be considered for 3.6 if the original issue can affect it too.

**Comment by lyuma** [comment]
Closing because #51591 was merged (default flag is now realtime).

I have filed a 3.x backport here: #62339 but I do not have a MRP, so I can't verify that it fixes anything.
